container/docker/handler.go

```diff
@@ -252,9 +252,11 @@ func newDockerContainerHandler(
 	// split env vars to get metadata map.
 	for _, exposedEnv := range metadataEnvs {
 		for _, envVar := range ctnr.Config.Env {
-			splits := strings.SplitN(envVar, "=", 2)
-			if splits[0] == exposedEnv {
-				handler.envs[strings.ToLower(exposedEnv)] = splits[1]
+			if envVar != "" {
+				splits := strings.SplitN(envVar, "=", 2)
+				if splits[0] == exposedEnv {
```

Comment:
I initially thought that too, but I tried removing it and didn't see any issues when passing values with multiple '='. It seems that SplitN only splits on the first occurrence of the delimiter, so omitting the ```len(splits) == 2```provides additional flexibility in that we can have equal signs in the environment variable values if we want to. I couldn't say whether that's useful or not for the particular reason behind this snippet?